index.html

```diff
@@ -9,37 +9,41 @@
 </head>
 <body>
     <div class="container">
-
         <div class="todo">
-            <header>
+            <header class="todo__header">
                 <h1>Todos</h1>
             </header>
 
-            <input type="text" name="new-todo" id="new-todo" placeholder="What needs to be done?">
-            <label class="small">
-                    <input type="checkbox" name="all-complete-todo" id="all-complete-todo">
-                    Mark all as complete
-            </label>
+            <div class="todo__content">
+                <input type="text" name="todo__new-todo-field" class="todo__new-todo-field" placeholder="What needs to be done?">
+
+                <div class="todo__complete-all-field checkbox-field checkbox-field--small">
+                    <label class="checkbox-field__label">
+                        <input type="checkbox" class="checkbox-field__input">
+                        Mark all as complete
+                    </label>
+                </div>
 
-            <div id="todo-items">
-                <ul>
-                    <li>
-                        <label>
-                            <input type="checkbox" name="todo-item-checkbox" class="todo-item-checkbox">
+                <ul class="todo-items">
+                    <li class="todo-item">
+                        <label class="todo-item__label">
+                            <input type="checkbox" name="todo-item__complete-checkbox" class="todo-item__complete-checkbox">
                             first
                         </label>
+                        <a class="todo-item__destroy"></a>
                     </li>
-                    <li>
-                        <label>
-                            <input type="checkbox" name="todo-item-checkbox" class="todo-item-checkbox">
+                    <li class="todo-item">
+                        <label class="todo-item__label">
+                            <input type="checkbox" name="todo-item__complete-checkbox" class="todo-item__complete-checkbox">
                             first
                         </label>
+                        <button class="todo-item__destroy"></button>
                     </li>
                 </ul>
             </div>
 
-            <div id="todo-status">
-                <span class="todo-count">3</span> items left
+            <div class="todo__summary">
```

Comment:
So this will probably work better as a `todo-summary` Then all the children can be things like `todo-summary__total-count`